cmd/tools/vdoc/tests/testdata/newlines/main.comments.out

```diff
@@ -14,7 +14,8 @@ fn funky()
     - bar
     # test
     ########### deep test
-    #a shouldn't have a newline test
+    #a shouldn't have a newline
+    test
```

Comment:
Used suboptimal words in the todo with `fix tests`. 
Because it is not the test that needs to be corrected, but the code so that the test passes again.

Afaik the `.out` was correct and as the text says the word should not be on a new line.
If updating the test is fine for this PR, the behavior with line breaks should be investigated separately.